config/commands.go

```diff
@@ -3,16 +3,27 @@ package config
 import (
 	"fmt"
 	"os"
+	"sort"
 	"strings"
 
 	"github.com/pkg/errors"
 	"gopkg.in/yaml.v3"
 )
 
 func printValues(values Values) {
-	for key, value := range values {
-		fmt.Fprintf(os.Stderr, "%-18s %s\n", key+":", value)
+	// Provide a stable sort order for printed values
+	keys := make([]string, 0, len(values))
+	for k := range values {
+		keys = append(keys, k)
```

Comment:
open to suggestion here; I didn't find a way to do this in place for the original map.

Reply:
I think this is fine - it's nice and explicit what's going on and we're not necessarily resource constrained here